_sass/_core/typography.scss

```diff
@@ -6,6 +6,21 @@ body {
   font-size: $base-font-size;
 }
 
+html, body, div, span, applet, object, iframe,
```

Comment:
Normally I would agree, but in this instance, consolidating them makes sense and is more readable.